BUILDGUIDE.md

```diff
@@ -319,6 +319,12 @@ TLS 1.3 has been excluded due to the fact that the driver lacks full support. To
 
 `Switch.Microsoft.Data.SqlClient.EnableSecureProtocolsByOS`
 
+## Suppressing TLS security warning
+
+Console security warning using an insecure protocol lower than `TLS 1.2` could be skipped on SQL connections with `Encrypt = false`, by enabling the following AppContext switch on application startup:
+
+`Switch.Microsoft.Data.SqlClient.SuppressTLSWarning`
```

Comment:
I suggest changing the name to `SuppressInsecureTLSWarning` to emphasize that only Insecure TLS related warnings will be skipped.
```suggestion
`Switch.Microsoft.Data.SqlClient.SuppressInsecureTLSWarning`
```

Reply:
I agree with Javad's suggestion.